### DISCUSSION TITLE
Customize telegram notifications

### DISCUSSION THREAD

**Discussion by passarela** [grafana/grafana#51383] [Alerting]
Hello,

I'm having a hard time knowing if today it's possible to customize the alerts sent by telegram.

I would like to change the default template:
**Resolved**
**Firing**
![This is an image](https://i.imgur.com/gM25Y9s.png)

Telegram alerts are extremely polluted, is there any way to change it? make it cleaner?

If so could you show me how?

**Comment by passarela** [comment]
I've seen some users with the same question and they were not successful.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
